test/functional/specs/Personalization/C6364798.js

```diff
@@ -230,30 +229,41 @@ const simulateViewChangeForNonExistingView = async alloy => {
     }
   });
 
-  const noViewViewChangeRequest = networkLogger.edgeEndpointLogs.requests[4];
+  const noViewViewChangeRequest = networkLogger.edgeEndpointLogs.requests[3];
   const noViewViewChangeRequestBody = JSON.parse(
     noViewViewChangeRequest.request.body
   );
   // assert that no personalization query was attached to the request
   await t.expect(noViewViewChangeRequestBody.events[0].query).eql(undefined);
-  // assert that a notification call with xdm.web.webPageDetails.viewName and no personalization meta is sent
-  await flushPromiseChains();
-  const noViewNotificationRequest = networkLogger.edgeEndpointLogs.requests[5];
-  const noViewNotificationRequestBody = JSON.parse(
-    noViewNotificationRequest.request.body
-  );
   await t
-    .expect(noViewNotificationRequestBody.events[0].xdm.eventType)
-    .eql("decisioning.propositionDisplay");
+    .expect(
+      // eslint-disable-next-line no-underscore-dangle
+      noViewViewChangeRequestBody.events[0].xdm._experience.decisioning
+        .propositions
+    )
+    .eql([
+      {
+        scope: "noView",
```

Comment:
Thanks for bringing this up. I've made a different change so that the lowercase scope is only used in the view cache. https://github.com/adobe/alloy/pull/1036/commits/ca330320872539075641d75ece3800d593da44d6